app/assets/javascripts/angular/components/surveyOverview/surveyOverview.html

```diff
@@ -3,37 +3,37 @@
     ng-class="$ctrl.activeTab === 'questions' ? '' : 'shadow-3'"
 >
     <div
-        class="header br2 br--top flex justify-between bg-pivot-blue-darker"
+        class="header br2 br--top bg-pivot-blue-darker"
         ng-class="$ctrl.activeTab === 'questions' ? 'shadow-3' : ''"
     >
-        <div class="">
-            <div class="f3 pivot-white ph3 pt3 flex">
-                <a
-                    ui-sref="app.ministries.ministry.surveys({ surveyId: survey.id })"
-                >
-                    <div class="mr2" ng-include="$ctrl.chevronLeftIcon"></div>
-                </a>
+        <div class="f3 pivot-white ph3 pt3 flex">
+            <a
+                ui-sref="app.ministries.ministry.surveys({ surveyId: survey.id })"
+            >
+                <div class="mr2" ng-include="$ctrl.chevronLeftIcon"></div>
+            </a>
+            <span class="flex-auto">
                 {{ $ctrl.survey.title }}
-            </div>
+            </span>
+            <a
+                ui-sref="appWithoutMenus.previewSurvey({ surveyId: $ctrl.survey.id })"
+                target="_blank"
+                class="self-center f6 link ba ph2 pv1 ph3-ns pv2-ns white bold br2 dim ttu tracked"
```

Comment:
So I noticed when testing this locally that when not mobile, the preview link on hover goes completely blue and unseeable. I think something with the tachyon link class here is overriding the white on hover. Changing the white to our pivot-white fixes it to what it used to be.
```suggestion
                class="self-center f6 link ba ph2 pv1 ph3-ns pv2-ns pivot-white bold br2 dim ttu tracked"
```